core/storage/base_model/gae_models.py

```diff
@@ -109,17 +115,29 @@ def get_multi(cls, entity_ids, include_deleted=False):
 
     @classmethod
     def put_multi(cls, entities):
+        """Stores sequence of entities.
+
+        Args:
+            entities: sequence.
+
+        Return:
+            list with the stored keys.
+        """
         return ndb.put_multi(entities)
 
     def delete(self):
         super(BaseModel, self).key.delete()
 
     @classmethod
     def get_all(cls, include_deleted_entities=False):
-        """Returns a filterable iterable of all entities of this class.
+        """Gets iterable of all entities of this class.
+
+        Args:
+            include_deleted_entities: bool. If is True then entities that have
```

Comment:
bool. Whether to include entities that are marked as 'deleted' in the return value. Defaults to False.